### PR TITLE
Feature/redesign overzichtspagina implementeren

### PR DESCRIPTION
**Pull request by jochemvogel**
# Description

Closes #193 

Branch name: `feature/redesign-overzichtspagina-implementeren`

Design dat in Figma gemaakt is, geïmplementeerd. 

## Type of change

Please select the option that is relevant

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

# Checklist

- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] All the required documentation is added.
- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.

# Additional information

_[Provided extra information if necessary]_

### CODE REVIEW

File: src/app.css [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 	--color-turquoise: hsl(192, 80%, 80%);
 	--color-dark-blue: hsl(221, 43%, 16%);
-	--color-light-orange: hsl(13, 100%, 84%);
+	--color-light-orange: hsl(92, 30%, 64%);
```
Comment (jochemvogel) [inline src/app.css:9]: Hoe gaan we dit doen? Was iemand anders hier al mee bezig?

File: src/app.css [hunk 2]
```diff
@@ -16,11 +16,18 @@
 	--color-white: hsl(0, 0%, 100%);
 }
 
+#svelte {
```
Comment (jochemvogel) [inline src/app.css:19]: Iemand hier een beter oplossing voor? Het is in ieder geval wel nodig.

File: src/lib/GameList/GameListCard.svelte [hunk 3]
```diff
@@ -28,33 +29,37 @@
...
+		border-radius: 1.5em;
 		margin: 1em 0;
 		padding: 1em;
 		justify-content: space-between;
-	}
-
-	li.offline {
-		background-color: var(--color-grey);
-		color: var(--color-white);
+		filter: drop-shadow(0 0.2em 0.2em hsl(120, 46%, 89%));
```
Comment (jochemvogel) [inline src/lib/GameList/GameListCard.svelte:62]: Gaf een net wat leuker effect dan een `box-shadow`

Comment (sreen020) [reply src/lib/GameList/GameListCard.svelte:62]: Misschien handig dit in een variable zetten

**Review by sreen020 — APPROVED** [review]

**Pull request merged**